meilisearch/tests/index/create_index.rs

```diff
@@ -68,21 +68,20 @@ async fn create_index_with_gzip_encoded_request_and_receiving_brotli_encoded_res
     let parsed_response =
         serde_json::from_slice::<Value>(decoded.into().as_ref()).expect("Expecting valid json");
 
-    assert_eq!(parsed_response["taskUid"], 0);
```

Comment:
I had to remove this as it appears to be different locally and in the CI, so it's hard to know the actual taskUid here...

Reply:
Makes sense since the server is shared, thanks!